subsys/net/ip/dhcpv4.c

```diff
@@ -287,6 +287,9 @@ static uint32_t dhcpv4_update_message_timeout(struct net_if_dhcpv4 *dhcpv4)
 		timeout = 64;
```

Comment:
In commit "net: ip: dhcpv4: Add randomization to message interval", could you add information about the RFC to the commit body, it would useful information here. So something like this
```
net: ip: dhcpv4: Add randomization to message interval

Add +1/-1 second randomization to the timeout as per RFC 2131 chapter 4.1.
```

Reply:
I updated the commit message.